.changeset/wild-rabbits-arrive.md

```diff
@@ -0,0 +1,48 @@
+---
+'astro': minor
+---
+
+The virtual module `astro:config` is now stable, and it can be used without experimental flag, which is now removed:
+
+```diff
+// astro.config.mjs
+export default defineConfig({
+-  experimental: {
+-    serializeConfig: true
+-  }
+})
+```
+
+The virtual module exposes two sub-paths for controlled access to your configuration:
+
+- `astro:config/client`: exposes config information that is safe to expose to the client.
+- `astro:config/server`: exposes additional information that is safe to expose to the server, such as file/dir paths.
+
+To enable this new virtual module, add the `experimental.serializeManifest` feature flag to your Astro config:
+
+  ```js
+  // astro.config.mjs
+  import { defineConfig } from 'astro/config';
+  export default defineConfig({
+    experimental: {
+      serializeManifest: true,
+    },
+  });
+  ```
+
+Then, you can access the module in any file inside your project to import and use values from your Astro config:
+
+  ```js
+  // src/utils.js
+  import { trailingSlash } from 'astro:config/client';
+
+  function addForwardSlash(path) {
+    if (trailingSlash === 'always') {
+      return path.endsWith('/') ? path : path + '/';
+    } else {
+      return path;
+    }
+  }
+  ```
+
+For a complete overview, and to give feedback on this experimental API, see the [Serialized Manifest RFC](https://github.com/withastro/roadmap/blob/feat/serialised-config/proposals/0051-serialized-manifest.md).
```

Comment:
``````suggestion
If you were previously using this feature, please remove the experimental flag from your Astro config:

```diff
// astro.config.mjs
export default defineConfig({
-  experimental: {
-    serializeConfig: true
-  }
})
```

If you have been waiting for feature stabilization before using configuration imports, you can now do so.

Please see [the `astro:config` reference](https://docs.astro.build/en/my-feature/) for more about this feature.
``````